src/components/input-panel/spec-editor/index.tsx

```diff
@@ -22,6 +22,7 @@ export function mapDispatchToProps(dispatch: Dispatch<EditorActions.Action>) {
       formatSpec: EditorActions.formatSpec,
```

Comment:
Can we remove this now that we have the commands?

Reply:
@domoritz I think we can now remove `this.props.format` and this function too.

Reply:
Go ahead.